tests/attr/test_input_x_gradient.py

```diff
@@ -75,15 +78,18 @@ def _input_x_gradient_base_assert(
                     )
                 self.assertEqual(input.shape, attribution.shape)
         else:
```

Comment:
Can you change this else to `elif isinstance(attributions, Tensor) `?
This captures the only other possibility for attributions and would not need any isinstance checks below.